test/external-modules/jvm-crash/src/javaRestTest/java/org/elasticsearch/test/jvm_crash/TeePrintStream.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.test.jvm_crash;
+
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+/**
+ * Copies output to another {@code PrintStream}, as well as an {@code OutputStream}
+ */
+class TeePrintStream extends PrintStream {
```

Comment:
Out of curiosity: is this a typo, or does it have a meaning I don't know?

Reply:
Its from a unix command - https://www.man7.org/linux/man-pages/man1/tee.1.html

Reply:
TIL